behaviour/graql/language/define.feature

```diff
@@ -230,157 +271,79 @@ Feature: Graql Define Query
       | SPR |
 
 
-  Scenario: define abstract entity type creates a type
-    Given graql define
-      """
-      define animal sub entity, abstract;
-      """
-    Given the integrity is validated
-    When get answers of graql query
+  Scenario: define that a type is a subtype of itself throws
+    Then graql define throws
       """
-      match $x type animal; get;
+      define dog sub dog;
       """
-    Then concept identifiers are
-      |     | check | value  |
-      | ANI | label | animal |
-    Then uniquely identify answer concepts
-      | x   |
-      | ANI |
 
 
-  Scenario: define concrete subtype of abstract entity creates child of its parent type
+  Scenario: define multiple copies of the same 'plays' creates a type that plays that role
```

Comment:
we like the term "idempotent"

Reply:
Renamed to `define 'plays' is idempotent`